agenda-webapps/src/main/webapp/vue-app/agenda/components/event/AgendaEventDialog.vue

```diff
@@ -82,7 +95,6 @@ export default {
       }
     });
     this.$root.$on('agenda-event-details', agendaEvent => {
-      this.isForm = false;
```

Comment:
Why deleting this ?

Reply:
i think it was mistake from the other branch, because other branch not uptodate withe feature/agenda branch

Reply:
i don't change any thing https://github.com/exoplatform/agenda/blob/ccf3a59c780a0c23eca152fb9483665ae74c3f06/agenda-webapps/src/main/webapp/vue-app/agenda/components/event/AgendaEventDialog.vue#L61

Reply:
No need for deleting this but, you delete it when we add the redirection for the deatils view